index.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
+
 var fs = require('fs');
 var path = require('path');
 var crypto = require('crypto');
+var gulp = require('gulp');
 var gutil = require('gulp-util');
 var through = require('through2');
 
-// ignore missing file error
+/**
+ * Name of this gulp plugin.
+ *
+ * @type {string}
+ */
+var pluginName = 'gulp-changed';
```

Comment:
`const` is discouraged in strict mode and the name might be needed more than once in the code and you don't want to repeat yourself (DRY).